rollup.config.mjs

```diff
@@ -14,12 +14,24 @@ import dts from 'rollup-plugin-dts';
 import jscc from 'rollup-plugin-jscc';
 import { visualizer } from 'rollup-plugin-visualizer';
 
+/** @typedef {import('rollup'              ).RollupOptions                } RollupOptions                 */
+/** @typedef {import('rollup'              ).Plugin                       } Plugin                        */
+/** @typedef {import('rollup'              ).OutputOptions                } OutputOptions                 */
+/** @typedef {import('@rollup/plugin-babel').RollupBabelInputPluginOptions} RollupBabelInputPluginOptions */
+/** @typedef {import('@rollup/plugin-strip').RollupStripOptions           } RollupStripOptions            */
```

Comment:
/** @typedef {import('rollup').RollupOptions} RollupOptions */
/** @typedef {import('rollup').Plugin} Plugin */
/** @typedef {import('rollup').OutputOptions} OutputOptions */
/** @typedef {import('@rollup/plugin-babel').RollupBabelInputPluginOptions} RollupBabelInputPluginOptions */
/** @typedef {import('@rollup/plugin-strip').RollupStripOptions} RollupStripOptions */

Reply:
I know this is visually pleasing but we don't align imports/typedefs/etc like there elsewhere in the codebase, so for the purpose of standardization, I've suggested the above.